WordPress/UITestsFoundation/Globals.swift

```diff
@@ -53,6 +53,23 @@ public func waitAndTap( _ element: XCUIElement, maxRetries: Int = 10) {
     }
 }
 
+public func tapUntilExpectedElementAppear( toTap elementToTap: XCUIElement, toAppear elementToAppear: XCUIElement, maxRetries: Int = 10) {
```

Comment:
Nitpick: missing `s` in `...Appear` and an extra space before `toTap`:

```suggestion
public func tapUntilExpectedElementAppears(toTap elementToTap: XCUIElement, toAppear elementToAppear: XCUIElement, maxRetries: Int = 10) {
```

P.S. Since both function name and argument name are descriptive, you can use somehting like:

```suggestion
public func tap(element: XCUIElement, untilAppears elementToAppear: XCUIElement, times: Int = 10) {
```

Reply:
thanks for the suggestion @pachlava! i'll update it to be just `tap` with those arguments, it reads better that way too. updated in https://github.com/wordpress-mobile/WordPress-iOS/pull/21704/commits/f99cd29b6f654e6bd286d031f00b90c142ee577a